spec/unit/provider_resolver_spec.rb

```diff
@@ -511,10 +511,11 @@ def resource_class(resource)
 
       supported_providers = [
         :apt_package, :bash, :breakpoint, :chef_gem, :cookbook_file, :csh, :deploy,
-        :deploy_revision, :directory, :dpkg_package, :easy_install_package,
-        :erl_call, :execute, :file, :gem_package, :git, :http_request, :link, :log, :pacman_package, :paludis_package,
-        :perl, :python, :remote_directory, :route, :rpm_package, :ruby, :ruby_block, :script,
-        :subversion, :template, :timestamped_deploy, :whyrun_safe_ruby_block, :yum_package, :homebrew_package,
+        :deploy_revision, :directory, :dpkg_package, :easy_install_package, :erl_call,
+        :execute, :file, :gem_package, :git, :homebrew_package, :http_request, :link,
+        :log, :macports_package, :pacman_package, :paludis_package, :perl, :python,
+        :remote_directory, :route, :rpm_package, :ruby, :ruby_block, :script, :subversion,
+        :template, :timestamped_deploy, :whyrun_safe_ruby_block, :yum_package,
```

Comment:
I moved `:homebrew_package` and `:macports_package` to their correct alphabetical order and made the lines roughly the same length.